sea-query-binder/src/sqlx_postgres.rs

```diff
@@ -105,6 +107,10 @@ impl sqlx::IntoArguments<'_, sqlx::postgres::Postgres> for SqlxValues {
                 Value::TimeDateTimeWithTimeZone(t) => {
                     args.add(t.as_deref());
                 }
+                #[cfg(feature = "postgres-interval")]
+                Value::Interval(t) => {
+                    args.add(t.as_deref());
```

Comment:
How would I implement `sqlx::Encode<'_, Postgres>` for `PgIntervalValue` without adding sqlx as a dependency to `sea-query`?